### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by anoraktrend**

### CODE REVIEW

**Comment by xplshn** [comment]
Why is `python` a part of `base`? I thought Iglunix strived for Unix style minimalism and efficiency.

**Comment by anoraktrend** [comment]
Necessary for building other packages.  No other reason.

File: linux/linux/CONFIG.x86_64
Comment (Ella-0) [inline linux/linux/CONFIG.x86_64:1]: This file should be removed. we deliberately don't maintain our own kernel config. if we can not support an option we disable it manually in build.sh

File: gui/weston/build.sh
Comment (Ella-0) [inline gui/weston/build.sh:1]: weston should probably be removed tbh

File: gui/seatd/build.sh [hunk 1]
```diff
@@ -1,6 +1,11 @@
 pkgname=seatd
 pkgver=0.8.0
+<<<<<<< HEAD
+pkgrel=1
 mkdeps=samurai:muon:pkgconf
+=======
+mkdeps=samurai:meson:pkgconf
+>>>>>>> updates
```
Comment (Ella-0) [inline gui/seatd/build.sh:8]: merge fail

File: gui/ribidi/build.sh
Comment (Ella-0) [inline gui/ribidi/build.sh:1]: drop these changes; ribidi hasn't had a release yet and isn't really used by anything; we patch out bidi support. If you need me to make a release of ribidi make an issue on that repo though some work is needed on that project first.

File: gui/libudev-zero/build.sh
Comment (Ella-0) [inline gui/libudev-zero/build.sh:1]: why these changes?

File: gui/libdisplay-info/build.sh
Comment (Ella-0) [inline gui/libdisplay-info/build.sh:1]: NO we deliberately do not package this.

File: gui/libass/build.sh
Comment (Ella-0) [inline gui/libass/build.sh:1]: yh lol this package is broken af rn. should probably drop it

File: gui/fontconfig/build.sh
Comment (Ella-0) [inline gui/fontconfig/build.sh:1]: uhhh can you confirm if the stuff todo with fcobjshash gnu requirements has been resolved?

File: gui/basu/build.sh
Comment (Ella-0) [inline gui/basu/build.sh:1]: why the C flags changes and we're probably gonna drop this package in favour of some either v4l2 loopback or a new pipewire spa plugin for screen capture and some changes to libwebrtc

File: gnome/glib/build.sh
Comment (Ella-0) [inline gnome/glib/build.sh:1]: does download.gnome.org not have newer versions?

File: gnome/dbus/build.sh
Comment (Ella-0) [inline gnome/dbus/build.sh:1]: no we don't need to specify our own custom system bus socket

File: extra/lua/LICENSE
Comment (Ella-0) [inline extra/lua/LICENSE:1]: UHHHHHH NO???? we can not claim that I wrote lua?

File: extra/go/build.sh
Comment (Ella-0) [inline extra/go/build.sh:1]: probably gonna drop this in the future

File: extra/glow/build.sh
Comment (Ella-0) [inline extra/glow/build.sh:1]: probably gonna drop this in the future but haven't found an alternative not using go yet

File: base/hwdata/build.sh
Comment (Ella-0) [inline base/hwdata/build.sh:1]: no? why are we removing this? we use this in toybox for lspci, lsusb. we also use this in sway/wlroots instead of libdisplay-info

File: gui/seatd/build.sh (hunk 1 above)
Comment (anoraktrend) [reply gui/seatd/build.sh:8]: Yeah I see.

File: extra/lua/LICENSE
Comment (anoraktrend) [reply extra/lua/LICENSE:1]: Temporary file error

File: gnome/glib/build.sh
Comment (anoraktrend) [reply gnome/glib/build.sh:1]: Unfortunately, no. I actually checked and a lot of their packages are behind.

File: gui/basu/build.sh
Comment (anoraktrend) [reply gui/basu/build.sh:1]: Cflags, no idea, I know the LDflags are needed because of changes to LD in newer versions of lld.

File: gui/fontconfig/build.sh
Comment (anoraktrend) [reply gui/fontconfig/build.sh:1]: I will probably uncomment that, I just didn’t want to deal with updating patches.

File: gui/ribidi/build.sh
Comment (anoraktrend) [reply gui/ribidi/build.sh:1]: The release is needed because it doesn’t package otherwise. The other option is to move ribidi into “experimental”

File: linux/linux/CONFIG.x86_64
Comment (anoraktrend) [reply linux/linux/CONFIG.x86_64:1]: It was added as a temporary file to share with you what a working config looks like.

File: gui/weston/build.sh
Comment (anoraktrend) [reply gui/weston/build.sh:1]: Possibly. It can work though, I also like not having a separate package for the terminal emulator.

File: base/hwdata/build.sh
Comment (anoraktrend) [reply base/hwdata/build.sh:1]: not entirely sure how this got dropped.

File: gnome/dbus/build.sh
Comment (anoraktrend) [reply gnome/dbus/build.sh:1]: fixed

File: base/hwdata/build.sh
Comment (anoraktrend) [reply base/hwdata/build.sh:1]: fixed

File: extra/glow/build.sh
Comment (anoraktrend) [reply extra/glow/build.sh:1]: Go is useful, (as it's a whole language). This no longer builds.

**Comment by anoraktrend** [comment]
I HAVE NO IDEA how to clean a commit history
